homework/insensitive-palindrom/palindromecheck.cpp

```diff
@@ -0,0 +1,16 @@
+#include "palindromecheck.hpp"
+
+bool is_palindrome(std::string Text)
+{
+    std::transform(Text.begin(), Text.end(), Text.begin(), [](auto character)
+
+                   { return std::toupper(character); });
+    Text.erase(std::remove_if(Text.begin(), Text.end(), [](auto character)
+                              { return character < 'A' || character > 'Z'; }),
+               Text.end());
```

Comment:
Better remove characters then tranform them (you will have less elemnts to tranform :))